fili-core/src/main/java/com/yahoo/bard/webservice/web/filters/HealthCheckFilter.java

```diff
@@ -64,4 +81,15 @@ public void filter(ContainerRequestContext requestContext) throws IOException {
                 .filter(entry -> !entry.getValue().isHealthy())
                 .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
     }
+
+    /**
+     * Render the URI as a string.
+     *
+     * @param uri The URI to render
+     *
+     * @return A String representation of the URI
+     */
+    protected String renderUri(URI uri) {
```

Comment:
why is it protected?

Reply:
I put it out of bard logging filter. No real reason to make it public. We can make the one in BardLoggingFilter public static if you want.